src/Plugins/ITKImageProcessing/src/ITKImageProcessing/Filters/Algorithms/ITKImportFijiMontage.cpp

```diff
@@ -0,0 +1,323 @@
+
+    if(!dimFound || !dataFound)
+    {
+      return;
+    }
+
+    // Example line to be parsed: slice_12.tif; ; (471.2965233276666, -0.522608066434236)
+    while(std::getline(m_InStream, line))
+    {
+      line = StringUtilities::trimmed(line);
+      if(line.empty())
+      {
+        continue;
+      }
+      std::vector<std::string> tokens = StringUtilities::split(line, ';');
+      if(tokens.size() != 3)
+      {
+        continue;
+      }
+      BoundsType bound;
+      std::stringstream path;
+      path << m_InputValues->inputFilePath.parent_path().string() << "/" << tokens[0];
+      bound.Filepath = fs::path(path.str());
+
+      std::string coords = StringUtilities::trimmed(tokens[2]);
+      coords = StringUtilities::replace(coords, "(", "");
+      coords = StringUtilities::replace(coords, ")", "");
+      tokens = StringUtilities::split(coords, ',');
+      if(tokens.size() != 2)
+      {
+        continue;
+      }
+      float32 x = std::stof(tokens[0]);
+      float32 y = std::stof(tokens[1]);
+      bound.Origin = Point3Df(x, y, 0.0f);
+      m_Cache.bounds.push_back(bound);
+    }
+  }
+
+  // -----------------------------------------------------------------------------
+  void FillCache()
+  {
+    // This next function will set the FileName (Partial), Row, Col for each "bound" object
+    ParseConfigFile();
+
+    Point3Df minCoord = {std::numeric_limits<float32>::max(), std::numeric_limits<float32>::max(), std::numeric_limits<float32>::max()};
+
+    // Get the meta information from disk for each image
+    for(auto& bound : m_Cache.bounds)
+    {
+      std::stringstream dcNameStream;
+      dcNameStream << m_InputValues->imagePrefix << bound.Filepath.filename().string().substr(0, bound.Filepath.filename().string().find(bound.Filepath.extension().string()));
+      bound.ImageName = dcNameStream.str();
+
+      {
+        minCoord[0] = std::min(bound.Origin[0], minCoord[0]);
+        minCoord[1] = std::min(bound.Origin[1], minCoord[1]);
+        minCoord[2] = 0.0f;
+      }
+    }
+
+    Point3Df overrideOrigin;
+    // Now adjust the origin/spacing if needed
+    if(m_InputValues->changeOrigin)
+    {
+      overrideOrigin = m_InputValues->origin;
+      Point3Df delta = {minCoord[0] - overrideOrigin[0], minCoord[1] - overrideOrigin[1], minCoord[2] - overrideOrigin[2]};
+      for(auto& bound : m_Cache.bounds)
+      {
+        std::transform(bound.Origin.begin(), bound.Origin.end(), delta.begin(), bound.Origin.begin(), std::minus<>());
+      }
+    }
+
+    std::stringstream ss;
+    ss << "\n"
+       << "Imported Image Count: " << m_Cache.bounds.size();
+    m_Cache.montageInformation = ss.str();
+  }
+
+  // -----------------------------------------------------------------------------
+  Result<> ReadImages()
+  {
+    Result<> outputResult = {};
+
+    auto* filterListPtr = Application::Instance()->getFilterList();
+    auto imageImportFilter = filterListPtr->createFilter(FilterTraits<ITKImageReader>::uuid);
```

Comment:
Since we're in the ITKImageProcessing plugin already, we should just be able to include the filter directly.